sections/404-error.liquid

```diff
@@ -0,0 +1,61 @@
+
+<header class="section-header section-header--image has-parallax" style="background-image: url({{section.settings.background-404 | img_url: '1000x' }}); transform: translate3d(0px, 0px, 0px);" data-parallax="true" data-speed="0.5" data-direction="down">
```

Comment:
the data-*, and transform are applied at run-time (I believe) so don't need to be added explicitly.  You can use the collection-template as an example. 

Also run the HTML and JSON through a formatter so the indentation is correct.